metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -276,10 +254,13 @@ def echo(msg, stream='stderr', batch_id=None):
             batch.launch_job(
                 step_name,
                 step_cli,
-                task_spec,
-                code_package_sha,
-                code_package_url,
-                ctx.obj.datastore.TYPE,
+                common_attrs.flow_name,
+                common_attrs.run_id,
+                common_attrs.task_id,
+                str(common_attrs.attempt),
```

Comment:
Minor minor nit - `flow_name`, `run_id`, `task_id`, `attempt` can come directly from kwargs and other variables rather than common_attrs. It's not immediately apparent if common_attrs is mutating `flow_name`, `run_id`, `task_id` and `attempt` or not.

Reply:
Any reason to not use`retry_count` instead of `common_attrs.attempt`?